en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -39,15 +39,15 @@ Ex: Enable landing page. **defaultTheme.json** ( Teant theming ).
 ```
 ### Tenant Theming 
 
-#### Uploading Devportal theme via the Admin Portal (Tenants Only)
+#### Uploading/Downloading Devportal theme via the Admin Portal (Tenants Only)
```

Comment:
Do we need to specifically mention about download?

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/1659/commits/04ddc59e90b732a370a7e4752bbbc7ac16f267f0